### PR TITLE
dark theme

### PR DESCRIPTION
**Pull request by tarunsankhla**

### CODE REVIEW

File: Documentation/Documentation.css [hunk 1]
```diff
@@ -50,13 +50,13 @@ body::-webkit-scrollbar:hover{
...
-.navbar-items{
+.navbar-items-main{
     text-decoration: none;
     color: var(--boldy-dark-purple);
     font-weight: 500;
 }
 
-a.navbar-items {
+a.navbar-items-main {
     padding: 1rem;
```
Comment (PuneetP16) [inline Documentation/Documentation.css]: The neat modular approach there!

File: Documentation/Documentation.css [hunk 2]
```diff
@@ -164,6 +166,8 @@ fieldset{
 .markup{
     background-color: var(--box-shadow);
     font-weight: 500;
+    padding: 0.3rem;
```
Comment (PuneetP16) [inline Documentation/Documentation.css]: Is it supposed to be in rems? my suggestion is to use px for anything below 10px as values in decimals seem like experimented values, also for such a low value, responsiveness isn't expected.

File: Documentation/Documentation.css [hunk 3]
```diff
@@ -412,14 +417,154 @@ aside{
...
+    color:  var(--box-shadow-dark);
+}
+.dark-mode .dialog{
+    color:  var(--box-shadow-dark);
+}
+
+.dark-mode .navbar-items-main{
+
+    background-color:  var(--dark-theme-bg);
+    color: var(--boldy-mid-gray);
```
Comment (PuneetP16) [inline Documentation/Documentation.css]: Uneven spaced could be fixed in here!

File: Documentation/Documentation.css [hunk 4]
```diff
@@ -412,14 +417,154 @@ aside{
...
+    background-color: var(--dark-theme-bg);
+} 
+
+.dark-mode .landing-container::before {
+    background-image: linear-gradient(-45deg, var(--btn-text-color) 0%, green 150%);
+
+}
+
+
+/* checkbox btn them */
```
Comment (PuneetP16) [inline Documentation/Documentation.css]: here too!

**Review by PuneetP16 — APPROVED** [review]
There some issue, there isn't change in background color on switching theme!
image

**Review by PuneetP16 — CHANGES_REQUESTED** [review]
By mistake approved it, can you fix the necessary to fix issue for approval.

**Review by Debarshi95 — APPROVED** [review]
Looks good. Can merge

File: Documentation/Documentation.css (hunk 2 above)
Comment (swethaogeti) [inline Documentation/Documentation.css]: for smaller values less than 1rem try using px

File: Documentation/Documentation.css [hunk 5]
```diff
@@ -412,14 +422,154 @@ aside{
...
+  
+  .fa-moon {
+    color: pink;
+  }
+  
+  .fa-sun {
+    color: yellow;
+  }
+
+
```
Comment (swethaogeti) [inline Documentation/Documentation.css]: be consistent in using var()

File: Documentation/Documentation.html [hunk 6]
```diff
@@ -22,30 +22,42 @@
     <script defer src="./script.js"></script>
     <script defer src="../Components/Drawer/Drawer.js"></script>
     <script defer src="../Components/Rating/Rating.js"></script>
+    <script defer src="../Documentation/theme.js"></script>
     <title>Tangent-UI</title>
 </head>
 <body onresize="alterClass()">
     <div class="navbar">
         <div class="navbar-title">
             <img loading="lazy" loading="lazy" class="navbar-image" src="../assets/Img/logo.png">
```
Comment (swethaogeti) [inline Documentation/Documentation.html]: alt tag can be added

File: index.html [hunk 7]
```diff
@@ -59,7 +61,9 @@ <h1>Installation</h1>
          <img class="installation-img" src="./assets/Img/homebackg.png" alt="">
```
Comment (swethaogeti) [inline index.html]: alt tag is left empty check once

File: index.html [hunk 8]
```diff
@@ -79,8 +83,8 @@ <h1>Installation</h1>
       </script>
     </div>
     <footer>
-        <div><a href="https://github.com/tarunsankhla/StyleComponent" class="btn btn=icon">Go to code</a></div>
-        Made with ❤ by Tarun.
+        <!-- <div><a href="https://github.com/tarunsankhla/StyleComponent" class="btn btn=icon">Go to code</a></div> -->
```
Comment (swethaogeti) [inline index.html:87]: change btn=icon class name

File: index.html [hunk 9]
```diff
@@ -79,8 +83,8 @@ <h1>Installation</h1>
...
+        Made with ❤ by <a class="footer-navigator" href="https://github.com/tarunsankhla/"> Tarun</a>.
```
Comment (swethaogeti) [inline index.html:88]: rest is good !! GTM

**Review by swethaogeti — APPROVED** [review]
check suggestions rest is good ! can merge

File: Documentation/Documentation.css (hunk 1 above)
Comment (tarunsankhla) [reply Documentation/Documentation.css]: done

File: Documentation/Documentation.css (hunk 2 above)
Comment (tarunsankhla) [reply Documentation/Documentation.css]: okay thanks for suggestion

Comment (tarunsankhla) [reply Documentation/Documentation.css]: okay done thanks for suggestion

File: Documentation/Documentation.css (hunk 3 above)
Comment (tarunsankhla) [reply Documentation/Documentation.css]: done

File: Documentation/Documentation.css (hunk 4 above)
Comment (tarunsankhla) [reply Documentation/Documentation.css]: done

File: Documentation/Documentation.css (hunk 5 above)
Comment (tarunsankhla) [reply Documentation/Documentation.css]: okay done

File: Documentation/Documentation.html (hunk 6 above)
Comment (tarunsankhla) [reply Documentation/Documentation.html]: done

File: index.html (hunk 7 above)
Comment (tarunsankhla) [reply index.html]: done

File: index.html (hunk 8 above)
Comment (tarunsankhla) [reply index.html:87]: removed that button completely

File: index.html (hunk 9 above)
Comment (tarunsankhla) [reply index.html:88]: thanks for reviewwing

**Comment by tarunsankhla** [comment]
> There some issue, there isn't change in background color on switching theme! image

I have fixed it now do check

**Pull request merged**